cg/cli/workflow/base.py

```diff
@@ -0,0 +1,20 @@
+"""Common CLI workflow functions"""
+
+
+import click
+
+from .balsamic.base import balsamic as balsamic_cmd
+from .microsalt.base import microsalt
+from .mip_dna.base import mip_dna
+from .mip_rna.base import mip_rna
+
+
+@click.group()
+def workflow():
```

Comment:
Seeing that all workflows are grouped in this command, I think it would make sense to instantiate some APIs used in all the workflow here and then pass them to context. For example, I think that Store and Housekeeper are used for all of them.

Reply:
Yes that might make sense in a way. I'm afraid well couple the workflows a bit to tightly if we require them to have those thing implicitly. I prefer the explicit way. What do you say?

Reply:
I see your point, if the workflows are to be worked on completely independent from each other, maybe this is the way to go 👍